backend/backend/services/openAI/service.py

```diff
@@ -18,16 +18,22 @@ def generate_prompt(input):
 
 
 class OpenAIService:
-  def __init__(self, use_prompt=0):
-    openai.api_key = OPENAI_API_KEY
+    def __init__(self, use_prompt=0):
+        openai.api_key = OPENAI_API_KEY
```

Comment:
I been thinking about this, it's nice to have it here for using any script, but its nice to have it happen at app start up in the flask app also